spring-integration-core/src/main/java/org/springframework/integration/config/IntegrationRegistrar.java

```diff
@@ -414,14 +423,91 @@ private void registerDefaultDatatypeChannelMessageConverter(BeanDefinitionRegist
 	 * @param registry the registry.
 	 */
 	private void registerArgumentResolverMessageConverter(BeanDefinitionRegistry registry) {
-		if (!registry.containsBeanDefinition(IntegrationContextUtils.ARGUMENT_RESOLVER_MESSAGE_CONVERTER_BEAN_NAME)) {
-			BeanDefinitionBuilder postProcessorBuilder = BeanDefinitionBuilder
+		if (!registry.containsBeanDefinition(
+				IntegrationContextUtils.ARGUMENT_RESOLVER_MESSAGE_CONVERTER_BEAN_NAME)) {
+			BeanDefinitionBuilder resolverBuilder = BeanDefinitionBuilder
```

Comment:
I agree that previous `postProcessorBuilder` name wasn't good, but still isn't `Resolver`. It is `MessageConverter`